ChangeLog.md

```diff
@@ -7,6 +7,14 @@
    the width (vertical/horizontal or Euclidean) of a convex hull.
    (Bertrand Kerautret, [#1052](https://github.com/DGtal-team/DGtal/pull/1052))
 
+ - ScaleProfile: new class to represent a (multi)scale profile i.e. a sequence
+   of statistics on digital lengths parameterized by a scale.
```

Comment:
I don't understand the "digital lengths".. anything could be stored in a ScaleProfile, right ?

BTW I have the feeling that the ScaleProfile name for this is confusing..